src/data/lovelace_custom_cards.ts

```diff
@@ -0,0 +1,23 @@
+export interface CustomCardEntry {
+  type: string;
+  name?: string;
+  description?: string;
+  preview?: boolean;
+}
+
+export interface CustomCardsWindow {
+  customCards?: CustomCardEntry[];
+}
+
+export const getCustomCards = () => {
+  return ((window as CustomCardsWindow).customCards || []).filter((card) => {
+    return card.type !== undefined;
```

Comment:
Why would we filter it ? Way better to let things blow up so custom card author knows where to look.

Reply:
Because anyone can change their mind I guess.
![Skärmavbild 2020-03-27 kl  07 06 34](https://user-images.githubusercontent.com/1299821/77727315-4e09e580-6ffa-11ea-9e5f-4feda89ee4dd.png)

I think you're right, though. Better to blow up.